### ISSUE TITLE
Dropping file onto .exe extracts contents in unknown output

### ISSUE THREAD

**Issue by ricolantern** [usagirei/RGSS-Decryptor#3]
Very annoying.
It extracted about 1gb worth of files in my C disk, but they're nowhere to be found.
A message telling you where your extracted files will be sent to would be nice, as I now have about 2gb remaining on my disk that is reserved for Windows itself.

**Comment by usagirei** [comment]
I don't know how you did it, but the files should be in the same place as
the rgss file:

https://github.com/usagirei/RGSS-Decryptor/blob/master/RgssDecrypter/Program.cs#L150

And the output folder is already displayed:

https://github.com/usagirei/RGSS-Decryptor/blob/master/RgssDecrypter/Program.cs#L98

**Comment by usagirei** [comment]
Just to complement the about, default windows behavior when dropping a file on the executable will pass the file full path, and set the working directory to the directory of that file, unless it's a shortcut with the working directory set to something else.

 So under most cases the files should be in the same place as the file you dropped on it

**Comment by ricolantern** [comment]
I see. I can't seem to find out why it's dropping all the files *somewhere* on my C disk even though I have nothing related to the decryptor or the game there.

Aside from this, every time I open the decryptor executable normally, it closes half a second after opening.
Do you happen to know what could be causing this? My antivirus doesn't seem to be picking this up either, so that can't be the issue.
Running the executable as administrator won't change anything either.

**Comment by usagirei** [comment]
There is no user interface, only command line, so opening it without any arguments/files to extract it'll just close on its own, its intended behaviour.

Try copying the rgss file in the same directory as the executable, and opening a command prompt and calling it from there like:

`RgssDecryptor game.rgssad`

or whathever the extension is from the file you need to extract, that should make the output be there in the same place too

EDIT: To open a command prompt in a folder, hold shift  and right click, and do "Open command window here" on windows 7/8, microsoft removed that option from windows 10, so you'll need to open a command prompt and navigate to the correct directory `cd <path to directory>` and possibly  `<drive letter>:`

**Comment by ricolantern** [comment]
Just tried to do this using the command prompt and placing the rgss3a file in the same folder as the executable. All files extracted, once again, *somewhere on my C disk*.

Is there maybe some preset output you set?
I'd really like to remove all these files, but I can't find them at all.
I've already tried looking for the most recent files on my pc, but these files created by your program don't show up.

**EDIT:**
Found the output folder.
It is, in fact, caused by some leftovers of a program I installed long ago, and believed to have uninstalled.
Can confirm that this issue is not an error in your program.

Thanks for helping me out, I'll try to take it from here!
